README.md

```diff
@@ -8,14 +8,14 @@
 This repository is a part of the Shotgun Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
-- For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
+- For general information and documentation, click here: https://developer.shotgridsoftware.com/e95205a8/?title=An+Overview+of+Toolkit
 - For information about Shotgun in general, click here: http://www.shotgunsoftware.com/toolkit
 
 ## Using this app in your Setup
 All the apps that are part of our standard app suite are pushed to our App Store.
 This is where you typically go if you want to install an app into a project you are
 working on. For an overview of all the Apps and Engines in the Toolkit App Store,
-click here: https://support.shotgunsoftware.com/entries/95441247.
+click here: https://developer.shotgridsoftware.com/6d10dedf/?title=Apps+and+Engines+Configuration+Reference
```

Comment:
I think we should use this URL instead: https://developer.shotgridsoftware.com/162eaa4b/?title=Pipeline+Integration+Components